KsApi/MockService.swift

```diff
@@ -86,7 +86,12 @@
     fileprivate let fetchMessageThreadResult: Result<MessageThread?, ErrorEnvelope>?
     fileprivate let fetchMessageThreadsResponse: [MessageThread]
 
+    /**
+     FIXME: Eventually combine `fetchProjectEnvelopeResult` and `fetchProjectPamphletEnvelopeResult` once all calls returning `Project` are using GQL.
```

Comment:
Is this accounted for in another ticket within the epic? We should make sure there is ticket to track.

Reply:
[Ticketed](NTV-219 - [SPIKE] Convert all v1/projects related calls over to GQL)